github/github.go

```diff
@@ -119,6 +132,24 @@ func (client *Client) ConfigFile(repo *github.Repository, ref string) (*Bulldoze
 		return nil, errors.Wrapf(err, "cannot unmarshal .bulldozer.yml for %s on %s", repo.GetFullName(), ref)
 	}
 
+	// Default update strategy
+	if bulldozerFile.UpdateStrategy == "" {
+		bulldozerFile.UpdateStrategy = UpdateStrategyDeferToPR
+	}
+
+	allowedUpdateStrategies := []UpdateStrategy{UpdateStrategyAlways, UpdateStrategyDeferToPR}
+	validStrategy := func() bool {
+		for _, valid := range allowedUpdateStrategies {
+			if bulldozerFile.UpdateStrategy == valid {
+				return true
+			}
+		}
+		return false
+	}()
+	if !validStrategy {
+		return nil, errors.Errorf("Invalid update strategy %#v", bulldozerFile.UpdateStrategy)
```

Comment:
include the list of valid strategies in the error message? `allowedUpdateStrategies`